modules/nf-core/shinyngs/app/environment.yml

```diff
@@ -2,4 +2,5 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::r-shinyngs=2.0.0
+  - bioconda::r-shinyngs=2.1.0
+  - conda-forge::r-base=4.4.2
```

Comment:
I assume this pin is there to fix the snapshots. 

Actually, I think we should just take the R version out of the version outputs from the modules. I hadn't appreciated when I wrote the modules that it would be disruptive, and I don't think there's a good reason to pin R otherwise. 

We will in due course have lockfiles in nf-core to cover the real use case for this sort of thing.